kratos_element_test/view/ui_builder.py

```diff
@@ -262,6 +258,30 @@ def _toggle_mohr_options(self):
             for w in widgets:
                 w.pack_forget()
 
+    def setup_mohr_coulomb_controls(self, params):
+        self.mohr_checkbox = tk.BooleanVar()
+        self.cohesion_var = tk.StringVar(value="3")
+        self.phi_var = tk.StringVar(value="4")
+        self._create_mohr_options(params)
+
+        if self.is_linear_elastic:
+            self.controller.set_mohr_enabled(False)
+            self.controller.set_mohr_mapping(None, None)
+            self.mohr_checkbox_widget.configure(state="disabled")
+
+        elif self.is_mohr_coulomb:
+            self.controller.set_mohr_enabled(True)
+            self.cohesion_var = tk.StringVar(value="3")
+            self.phi_var = tk.StringVar(value="4")
+
+            c_idx, phi_idx = self._parse_mc_indices()
```

Comment:
Do we need to specifically set the cohesion_var and phi_var, or is it only used for finding the c_idx and phi_idx? In that case we could just write:

```suggestion
            c_idx = 3
            phi_idx = 4
```

Reply:
I removed the duplicated lines, and hopefully this is ok now?